### PR TITLE
Bump dependencies.

### PR DESCRIPTION
**Pull request by bigkevmcd**
This pulls in the latest dependencies from Flux (including some significantly out-of-date ones).

### CODE REVIEW

File: tfctl/main.go [hunk 1]
```diff
@@ -99,9 +98,7 @@ func download(version, resource string) ([]byte, error) {
 }
 
 func newManager(kubeClient client.Client) (*ssa.ResourceManager, error) {
-	kubePoller := polling.NewStatusPoller(kubeClient, kubeClient.RESTMapper(), polling.Options{})
-
-	return ssa.NewResourceManager(kubeClient, kubePoller, ssa.Owner{
+	return ssa.NewResourceManager(kubeClient, nil, ssa.Owner{
```
Comment (foot) [inline tfctl/main.go:101]: poller seems unused? 👍

**Comment by bigkevmcd** [comment]
This bumps the dependencies and brings them up-to-date.

**Comment by bigkevmcd** [comment]
We want to land this so that we can bump weave-gitops to a tag of tf-controller too.

**Review by yitsushi — APPROVED** [review]
lgtm

**Review by yitsushi — APPROVED** [review]

**Pull request merged**